parser.ts

```diff
@@ -397,44 +391,36 @@ class Parser {
   }
 
   private command(): Cmd | null {
-    return tracer.spanSync('command', () => {
```

Comment:
These spans aren't super helpful and they were making the types hard to debug, so I just removed them here.